docs/advanced-scenarios.md

```diff
@@ -152,4 +152,177 @@ Google's `id_token` contains basic identity data like name and user ID, which al
   },
 ```
 
+### Setup and use translation / internationalization (i18n)
+
+#### Overview
+
+This example shows you how to use Luigi with a centrally implemented translation dictionary.
+
+Luigi Core provides a generic [API for I18N](luigi-core-api.md#luigii18n). We will write a custom translation provider that gathers translation strings from static files hosted either on Core or on micro-frontend side.
+
+#### Steps
+
+1. Create a custom translation function
+2. Configure `settings.customTranslationImplementation`
+3. Load translations before `Luigi.setConfig()`
+4. Add locale change listener after Luigi initialization
+5. Use translation as a node label
```

Comment:
```suggestion
```

Since the steps are described below, maybe it's redundant to write them two times.

Reply:
Ok, I thought it might be nice to have a kind of index.